src/pages/components/accordion/usage.mdx

```diff
@@ -4,20 +4,180 @@ description: The accordion component delivers large amounts of content in a smal
+
+<DoDontRow>
+<DoDont
+colMd={4} colLg={4}
+caption="Do: use the default icon alignment to keep accordion text aligned with other content on the page.">
+
+![Do: use the default icon alignment to keep accordion text aligned with other content on the page.](./images/accordion-alignment-do.png)
+
+</DoDont>
+<DoDont
+colMd={4} colLg={4}
+type="dont"
+caption="Don’t: place icons on the left and create misalignment with the other text on the page.">
+
+![Don’t: place icons on the left and create misalignment with the other text on the page.](./images/accordion-alignment-dont.png)
+
+</DoDont>
+</DoDontRow>
+
+#### Modifier
+
+In rare scenarios, the accordion can be modified to place the icon in front of the title for only tree-like purposes. However, most instances should use the default alignment and for any pure content or documentation purposes it is required to use the default alignment. Icon placement should be consistent throughout your experience.
+
+### Placement
+
+Accordions can be placed with main page content or placed inside of a container such as a side panel or tile.
+
+<Row>
+<Column colLg={8}>
+
+![accordion placement](images/accordion-placement-2.png)
+
+</Column>
+</Row>
+
+#### Grid placement
+
+When placing accordion on the 2x Grid the indented title aligns to the grid columns and the top and bottom borders hang in the gutter.
 
 <Row>
 <Column colLg={8}>
 
-![accordion example](images/accordion-usage-1.png)
+![accordion grid placement](images/accordion-placement-1.png)
 
 </Column>
 </Row>
+
+## Behavior
+
+### States
+
+The accordion component has two main states: **collapsed** and **expanded**. The chevron icon at the end of the accordion indicates which state the accordion is in. The chevron points down to indicate collapsed and up to indicate expanded.
+
+Accordions begin by default in the collapsed state with all content panels closed. Starting in a collapsed state gives the user a high level overview of the available information.
+
+<Row>
+<Column colLg={8}>
+
+![accordion states](images/accordion-state-1.png)
+
+</Column>
+</Row>
+
+A user can then independently expand each section of the accordion allowing for multiple sections to be open at once.
+
+<Row>
+<Column colLg={8}>
+
+![multiple sections expanded](images/accordion-state-2.png)
+
+</Column>
+</Row>
+
+In addition to the collapsed and expanded states, checkboxes also have interactive states for focus, hover, disabled, and skeleton. See the [style tab](/components/accordion/style) for more details.
+
+<Row>
+<Column colLg={8}>
+
+![accordion interactives states](images/accordion-state-3.png)
+
+</Column>
+</Row>
+
+### Click target
+
+Users can trigger a state change by clicking on the chevron or clicking anywhere in the entire header area.
+
+<Row>
+<Column colLg={8}>
+
+![accordion click targets](images/accordion-click-target.png)
+
+</Column>
+</Row>
+
+## Content
+
+### Main elements
+
+#### Title
```

Comment:
It seems that WAI ARIA recommends that the titles be treated as a specific heading level (1-6), would we want to cover that here or would it be better in a different area? 👀 

Reference: https://www.w3.org/TR/wai-aria-practices-1.1/#wai-aria-roles-states-and-properties

> Each accordion header button is wrapped in an element with role heading that has a value set for aria-level that is appropriate for the information architecture of the page.

Reply:
Would this just be part of the accessibility tab content?

Reply:
Seems more about implementation than ux usage. So I think the code tab or accessibility tab might be the better fit.

Reply:
Now that I'm thinking of it. How does a user set a heading value of type inside of a component?

Reply:
Ok I added in a blurb about this.